pkg/ui/bindata.go

```diff
@@ -85,7 +85,7 @@ package ui
 import (
 	"bytes"
 	"compress/gzip"
-	"fmt"
+	"fmt" //lint:ignore faillint auto-generated file, hence skipping
```

Comment:
Let's discuss if we want to modify this.

Reply:
WDYT @bwplotka ?
Should we edit the file or let it stay as it is ?

Reply:
I think we cannot ): But maybe we can filter this file from faillint. I think even faillint was having some feature for ignoring generating files 🤔 Let's check in failint repo - We did not update this project recently

Reply:
I think we could improve the `assets` Makefile target to add this line at the top of the file during the run. Then, it would be ignored accordingly. WDYT?

Reply:
I think failint should ignore generated files by default. I asked @Biswajitghosh98 to check the upstream, let's see. (:

Reply:
@bwplotka I was working on another issue, and this completely escaped from my mind. I'll do it first thing tomorrow morning. Sorry for the delay 😅

Reply:
> I think failint should ignore generated files by default. I asked @Biswajitghosh98 to check the upstream, let's see. (:

I agree with @GiedriusS 's suggestion, improving on the assets Makefile would work for us, since faillint doesn't skip files that do not adhere its comment naming convention - [link](https://github.com/thanos-io/thanos/pull/3881#issuecomment-802343531)